raglweb/__init__.py

```diff
@@ -288,7 +288,30 @@ def info():
 @app.route('/replay/<replay_hash>')
 def replay(replay_hash):
     db = _db_get()
-    cur = db.execute('SELECT filename FROM outcomes WHERE hash=:hash', dict(hash=replay_hash))
-    fullpath = cur.fetchone()['filename']
+    cur = db.execute('''
+        SELECT
+            filename,
+            p0.division as p0_division,
+            p1.division as p1_division,
+        FROM outcomes o
+        LEFT JOIN players p0 ON p0.profile_id = o.profile_id0
+        LEFT JOIN players p1 ON p1.profile_id = o.profile_id1
+        WHERE hash=:hash
+    ''', dict(hash=replay_hash))
+    row = cur.fetchone()
+
+    prefix = 'RAGL-S10-'
+    p0_division = row['p0_division']
+    p1_division = row['p1_division']
+    if p0_division == p1_division:
+        division_str = p0_division
+        division_str = division_str.replace('Masters', 'MASTER-')
+        division_str = division_str.replace('Minions Alpha', 'MINIONA-')
+        division_str = division_str.replace('Minions Bravo', 'MINIONB-')
```

Comment:
This might be different between seasons (`laddertools/ragl-s9.yml` and `laddertools/ragl-s10.yml` are typically different, names are arbitrary) and we may forget to the adjust it. How about something more generic, like, either do something simple like `.upper().replace(' ','')`, or maybe work out something with the common sub-strings.

Reply:
The general method would be something like:
```
words = division.split(' ').upper()
division_str = words[0]
if division_str.endswith('S'):
    division_str = division_str[:-1]
for word in words[1:]:
    if len(word) > 0:
        division_str += word[0]
```

Would something this complex be ok?

Reply:
Yeah I think that's fine; feel free to isolate it in a function.